build.gradle

```diff
@@ -725,6 +725,7 @@ dependencies {
 
     // not working: odd
     //testCompile group: 'org.jenkins-ci.plugins', name: 'testInProgress-client', version: '1.1'
+    implementation files('lib/Java/opencsv-2.1.jar')
```

Comment:
In the past, we've used apache commons for writing CSV files. Would that be sufficient here? Example: https://github.com/zfin/zfin/blob/7a2d9dd7eb6d9a0879366e465ffa457d9feb9d0a/source/org/zfin/datatransfer/ctd/CreateMeshChebiMappingFile.java#L73

Reply:
It would be nice to re-use that library but I have to add some comments to the header part which is not the  header columns and I don't know how to add those lines with this CSV writer. Once I figure that out I can switch to use that library as well.